bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableBufferedMutator.java

```diff
@@ -178,18 +178,18 @@ public BigtableBufferedMutator(
       BufferedMutator.ExceptionListener listener,
       HeapSizeManager heapSizeManager) {
     this(client, adapter, configuration, options, listener, heapSizeManager,
-        createExecutorService(options));
+        createExecutorService(options), true);
   }
 
-  @VisibleForTesting
-  BigtableBufferedMutator(
+  public BigtableBufferedMutator(
```

Comment:
Is this constructor for testing only? if yes, annotate it with @VisibleForTesting